### PR TITLE
Allow viewing/restoring deleted pages for all users

### PR DESCRIPTION
**Pull request by ghost**
Not sure why this didn't class itself as a pull.

### CODE REVIEW

File: RWSettings.php [hunk 1]
```diff
@@ -380,6 +380,12 @@
 $wgGroupPermissions['tech']['gadgets-definition-create'] = true;
 $wgGroupPermissions['tech']['gadgets-definition-delete'] = true;
 
+#All users can view/restore deleted pages
+$wgGroupPermissions['*']['deletedhistory'] = true;
+$wgGroupPermissions['*']['deletedtext'] = true;
+$wgGroupPermissions['*']['undelete'] = true;
```
Comment (tstarling) [inline RWSettings.php:386]: The problem is that this also allows history merges. I will comment on the Saloon Bar.

**Review by tstarling — COMMENTED** [review]
Author notes on Saloon Bar "It's just a pull request -- I'm waiting for consensus here."

**Comment by Christopher36** [comment]
Pretty much everyone hated this idea, can this be deleted now?

**Comment by ghost** [comment]
burn with fire

**Pull request closed without merging**